client/packages/invoices/src/Returns/modals/OutboundReturn/ReturnSteps.tsx

```diff
@@ -38,13 +46,24 @@ export const ReturnSteps = ({
     return step === -1 ? 0 : step;
   };
 
+  const alertMessage =
+    zeroQuantityAlert === 'warning'
+      ? t('messages.warn-zero-return-quantity')
```

Comment:
NIT 😁 Might be easier as a dev to work out what was going on here if this one was something like `messages.zero-return-quantity-will-delete-lines`?